txn.go

```diff
@@ -280,6 +290,16 @@ func (txn *Txn) commitOrAbort(ctx context.Context) error {
 		return nil
 	}
 
+	txn.context.Keys = make([]string, 0, len(txn.keys))
+	for key := range txn.keys {
+		txn.context.Keys = append(txn.context.Keys, key)
```

Comment:
@NamanJain8 I think we can use map on dgraph side as well. We don't have to convert the map to slice

Reply:
We have decided not to do that because that will be breaking for `dgo` client. We can simply add deduplication on alpha.